oak_functions_service/proto/oak_functions.proto

```diff
@@ -28,8 +28,29 @@ service OakFunctions {
   rpc Invoke(InvokeRequest) returns (InvokeResponse);
 
   // Builds or replaces the existing lookup data with the new value.
+  // The value has to be below 2 GiB.
+  //
   // method_id: 2
   rpc UpdateLookupData(UpdateLookupDataRequest) returns (UpdateLookupDataResponse);
```

Comment:
@tiziano88 Should we keep or remove `UpdateLookupData`?

Reply:
We should remove it, once it's safe to do so (i.e. when we have a path for the update of the code in google3). If it's already possible to remove it now, then go ahead.